charts/pocket/pocket-servicer-overrides.yaml

```diff
@@ -0,0 +1,8 @@
+config:
+  servicer:
+    enabled: true
+    address: "001022b138896c4c5466ac86b24a9bbe249905c2"
+    public_key: "56915c1270bc8d9280a633e0be51647f62388a851318381614877ef2ed84a495"
```

Comment:
nit: add comments describing names of these values or linking to where more details can be found.

Reply:
Done, thank you for the review.